roles/foreman_installer/README.md

```diff
@@ -0,0 +1,28 @@
+theforeman.operations.foreman_installer
+======================================
+
+Run the foreman-installer
+
+Role Variables
+--------------
+
+- `foreman_installer_options`: Array of options to pass to the installer
+- `foreman_installer_verbose`: Enables verbose output mode in the installer
+- `foreman_installer_no_colors`: Disables color output from the installer
+- `foreman_installer_disable_system_checks`: Passes --disable-system-checks to the installer
+- `foreman_installer_skip_installer`: Skips running the installer
+- `foreman_installer_command`: Installer command to run, can be used by derivative projects to specify a branded command
+
+Example Playbooks
+-----------------
+
+Run the installer setting the initial admin password:
+
+```yaml
+- hosts: target-host
+  roles:
+    - role: theforeman.operations.foreman_installer
+      vars:
+        foreman_installer_options:
+          - '--foreman-initial-admin-password changeme'
```

Comment:
from a user POV, these are the things that I want to modify most often:
* admin-user
* admin-password
* primary org
* primary loc
* list of enabled plugins

I see plugins might be a bit tough as they don't have a stable naming in the cli options (partially because what users think are plugins are actually core features that can be disabled [like vmware] vs plugins [azure]).

Reply:
Updated the README with some of what you describe here -- is this what you are looking for?

Reply:
What I wanted to say (but obviously didn't): I think those are the ones that a pure user would love to see as individual options/variables/flags instead of passing around an array of cli options

Reply:
Oh, ha. Yes, one for each plugin would be harder to maintain. I'm happy to add the other 4 though they don't apply to every scenario (foreman-proxy-content, capsule) so that would be my only hesitation.

Reply:
all plugin enable options are of the form `--enable-foreman-plugin-PLUGIN_NAME`, would it be in a users favor if we provide a `enable_plugins` list (empty by default), which will translate into `--enable-foreman-plugin-PLUGIN_NAME` params?

I am not sure myself, so mostly throwing out ideas. Especially as removing the plugin from that list won't disable it…

Reply:
My leaning here is file an issue for re-evaluation but I want to start with the most basic version of this before building logic into it.

Reply:
ACK